fhir-search/src/main/java/com/ibm/fhir/search/compartment/CompartmentHelper.java

```diff
@@ -154,14 +154,24 @@ public static final void buildMaps(Map<String, CompartmentCache> compMap, Map<St
         return compartmentMap.get(compartment).getParametersByResourceTypeInCompartment(resourceType);
     }
 
+    /**
+     * checks that the compartment is valid and throws an exception if not
+     *
+     * @param compartment
+     * @throws FHIRSearchException
```

Comment:
This doesn't appear to throw any Exception, so the java doc should be updated.

Reply:
fixed